### PR TITLE
Migrate to conventional-changelog v6 + conventional-recommended-bump v10

### PR DESCRIPTION
**Pull request by webpro**
So let's get this going, it was long overdue.

First of all, bigshout out to @FRSgit who made ball roll in #97. And since I totally agree the options in this plugin should as much be passed through verbatim, I followed up his PR with this one that does just that.

Would be great to get this started and hear your thoughts, API feedback, bugs, etc!

This is under development in the [v9 branch](https://github.com/release-it/conventional-changelog/tree/v9) and updated docs are at https://github.com/release-it/conventional-changelog/blob/v9/README.md

The latest version in this branch can be installed using the command as provided by the pkg-pr-new bot below (`npm i -D`).

cc @will-stone, @skycaptain, @juancarlosjr97, @ardakod, @timmywil, @Blackclaws, @ygrishajev, @Blackclaws

**NOTE:** I could use some help maintaining this plugin. The thing is, I don't really use it myself. So not to spam everyone too much again later on I'm bringing that in here as well. Free free to respond or DM me about this.

### CODE REVIEW

**Comment by will-stone** [comment]
Hey, sorry I’ve had the notification in my inbox for ages from #88. I am hoping I can get some time to look into this but, having become a father this year, my free coding time is scarce 😅

File: README.md [hunk 1]
```diff
@@ -213,6 +232,26 @@ module.exports = {
...
+
+This option will be passed as the first argument to
+[`bumper.commits`](https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-recommended-bump/README.md#api)
+
+### `bumperParserOpts`
+
+This option will be passed as the second argument to
+[`bumper.parserOptions`](https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-recommended-bump/README.md#api)
+
+### `whatBump`
```
Comment (FRSgit) [inline README.md:250]: Nitpick:
If you're going with `bumper` prefix for other options (to scope them to underlying `bumper` package in a way, right?) then I think this one should be called:
```suggestion
### `bumperWhatBump`
```
just to follow the convention 🤷

**Comment by FRSgit** [comment]
Looks good! I've just updated version of the plugin on my monorepo and release process passed successfully! 🎉 

Wrote just a single nitpick comment on the naming of `whatBump` property. But either way - from my perspective it looks ready to be released. Great work!

**Review by FRSgit — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (webpro) [reply README.md:250]: Agreed, actually prefer unprefixed but then we'd have `parserOpts` 🙄 but they might actually be the same thing in conventional-changelog land, I'll double-check that.

**Comment by webpro** [comment]
The options are streamlined a bit and documentation is better structured now (i.e. options are listed in order of programmatic flow): https://github.com/release-it/conventional-changelog. I've also looked up type definitions in the conventional-changelog repos and added links.

The latest release is `@release-it/conventional-changelog@next` (9.0.0-next.0) and the idea is to publish v9 in a few days, depending on feedback. Feel free to discuss etc! Still open to any sort of ideas, feedback, etc.

Merging the branch closed this which wasn't really the intention, you can still comment!

**Comment by webpro** [comment]
v9 is out!

**Pull request merged**